app/controllers/players_controller.rb

```diff
@@ -6,6 +6,10 @@ def create
 
     if @instance.blank?
       redirect_to root_path, alert: "Lobby not found"
+    elsif @instance.status == 'ongoing'
+      redirect_to root_path, alert: "Game already started, try not to miss the next one."
+    elsif @instance.status == 'done'
+      redirect_to root_path, alert: "Game already finihsed, join the next one."
```

Comment:
should be "finished" not "finihsed" 😆

Reply:
Thanks for the catch, apologies for I dunno how to type haha.